index.html

```diff
@@ -43,6 +43,7 @@ <h3 id=basic>Basic Personal Website Setup</h3>
             <li><a href="dns.html">Set up DNS settings to connect your server and domain name.</a></li>
             <li><a href="nginx.html">Set up your web server.</a></li>
             <li><a href="certbot.html">Get a secure HTTPS connection with Certbot.</a></li>
+		  <li><a href="wildcard.html">Get a wildcard certificate using Certbot.</a></li>
```

Comment:
The indentation is off here. It should probably be consistent with all the other menu entries.